bin/config-yargs.js

```diff
@@ -14,53 +14,46 @@ module.exports = function(yargs) {
 		.alias("help", "h")
 		.version()
 		.alias("version", "v")
+		// These commands are not actually handled by yargs, as webpack.js
+		// pre-emptively looks for them prior to yargs processing. This is
+		// strictly for --help output
+		.command(
+			"init",
+			"Initializes a new webpack configuration or loads a addon if " +
+			"specified",
+		)
+		.command(
+			"migrate",
+			"Migrate your webpack configuration from webpack 1 to webpack 2"
```

Comment:
Better generalize this rather than specify a version to migrate from. Better rephrase to: "Migrate your webpack configuration to the newest version".